core/app/models/spree/address.rb

```diff
@@ -18,19 +18,41 @@ class Address < Spree::Base
     alias_attribute :first_name, :firstname
     alias_attribute :last_name, :lastname
 
+    UNCOPIED_ATTRS = ['id', 'updated_at']
```

Comment:
Why not include 'created_at' in this list also?  If someone is trying to set both created_at and updated_at why let them only change created_at and not update_at?